dandi/dandiapi.py

```diff
@@ -293,6 +293,22 @@ def dandi_authenticate(self):
                     lgr.debug("Stored key in keyring")
                 break
 
+    @property
+    def instance_name(self) -> str:
+        url = self.api_url.rstrip("/")
+        try:
+            return known_instances_rev[url]
+        except KeyError:
+            return url
+
+    @property
+    def upper_instance_name(self) -> str:
+        url = self.api_url.rstrip("/")
+        try:
+            return known_instances_rev[url].upper()
+        except KeyError:
+            return url
```

Comment:
```suggestion
    @property
    def _instance_id(self) -> str:
        url = self.api_url.rstrip("/")
        try:
            return known_instances_rev[url].upper()
        except KeyError:
            return url
```
and use that protected `_instance_id`  . Rationale:

- we use `instance_name`. Here it might also be a url, so a different beast, let's not occlude
- do not see a need for it to be "public" for now, since we might want to change what it is later on etc

Reply:
Done.